build error with cmake 4 and bundled libzmq

```
Building wheels for collected packages: pyzmq
  Building wheel for pyzmq (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pyzmq (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [567 lines of output]
      *** scikit-build-core 0.11.0 using CMake 4.0.0 (wheel)
      *** Configuring CMake...
      loading initial cache file /tmp/tmpziu6b28p/build/CMakeInit.txt
      -- The C compiler identification is GNU 15.0.1
      -- The CXX compiler identification is GNU 15.0.1
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /opt/gcc-latest/bin/gcc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /opt/gcc-latest/bin/g++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Found Python: /home/cc/venv/bin/python3.14t (found version "3.14.0") found components: Interpreter Development.Module
      -- Looking for libzmq
      -- Looking for libzmq with pkg-config
      -- Checking for module 'libzmq'
      --   Package 'libzmq' not found
      -- Looking for libzmq with pkg-config - no
      --   Fallback: looking for libzmq in /home/cc/venv/lib/python3.14t/site-packages;/tmp/pip-build-env-si4y4agd/overlay/lib/python3.14t/site-packages;/usr/local;/usr
      -- Looking for libzmq - libzmq not found, will bundle libzmq and libsodium
      -- Bundling libzmq and libsodium
      -- building bundled libsodium
      './configure' '--prefix=/tmp/tmpziu6b28p/build/bundled' '--with-pic' '--disable-dependency-tracking' '--disable-shared' '--enable-static'
      checking build system type... x86_64-pc-linux-gnu
      checking host system type... x86_64-pc-linux-gnu
      checking target system type... x86_64-pc-linux-gnu
      checking for a BSD-compatible install... /usr/bin/install -c
      checking whether build environment is sane... yes
      checking for a race-free mkdir -p... /usr/bin/mkdir -p
      checking for gawk... gawk
      checking whether make sets $(MAKE)... yes
      checking whether make supports nested variables... yes
      checking whether UID '1000' is supported by ustar format... yes
      checking whether GID '1000' is supported by ustar format... yes
      checking how to create a ustar tar archive... gnutar
      checking whether make supports nested variables... (cached) yes
      checking whether to enable maintainer-specific portions of Makefiles... no
      checking for gcc... gcc
      checking whether the C compiler works... yes
      checking for C compiler default output file name... a.out
      checking for suffix of executables...
      checking whether we are cross compiling... no
      checking for suffix of object files... o
      checking whether the compiler supports GNU C... yes
      checking whether gcc accepts -g... yes
      checking for gcc option to enable C11 features... none needed
      checking whether gcc understands -c and -o together... yes
      checking whether make supports the include directive... yes (GNU style)
      checking dependency style of gcc... none
      checking dependency style of gcc... none
      checking for stdio.h... yes
      checking for stdlib.h... yes
      checking for string.h... yes
      checking for inttypes.h... yes
      checking for stdint.h... yes
      checking for strings.h... yes
      checking for sys/stat.h... yes
      checking for sys/types.h... yes
      checking for unistd.h... yes
      checking for wchar.h... yes
      checking for minix/config.h... no
      checking whether it is safe to define __EXTENSIONS__... yes
      checking whether _XOPEN_SOURCE should be defined... no
      checking whether C compiler accepts -O3... yes
      checking for a sed that does not truncate output... /usr/bin/sed
      checking how to run the C preprocessor... gcc -E
      checking for egrep -e... /usr/bin/grep -E
      checking whether gcc is Clang... no
      checking whether pthreads work with "-pthread" and "-lpthread"... yes
      checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
      checking whether more special flags are required for pthreads... no
      checking for PTHREAD_PRIO_INHERIT... yes
      checking for variable-length arrays... yes
      checking for __wasi__ defined... no
      checking whether C compiler accepts -Werror... yes
      checking whether to add -D_FORTIFY_SOURCE=3 to CPPFLAGS... yes
      checking whether C compiler accepts -fvisibility=hidden... yes
      checking whether C compiler accepts -fPIC... yes
      checking whether C compiler accepts -fPIE... yes
      checking whether the linker accepts -pie... yes
      checking whether C compiler accepts -fno-strict-aliasing... yes
      checking whether C compiler accepts -fno-strict-overflow... yes
      checking whether C compiler accepts -fstack-protector... yes
      checking whether the linker accepts -fstack-protector... yes
      checking whether C compiler accepts  -O3 -pthread -fvisibility=hidden -fPIC -fPIE -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wall... yes
      checking whether C compiler accepts  -O3 -pthread -fvisibility=hidden -fPIC -fPIE -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations... yes
      checking whether C compiler accepts  -O3 -pthread -fvisibility=hidden -fPIC -fPIE -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas... yes
      checking for clang... no
      checking whether C compiler accepts  -O3 -pthread -fvisibility=hidden -fPIC -fPIE -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wall -Wextra... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wshorten-64-to-32... no
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wsometimes-uninitialized... no
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wstrict-prototypes... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wstrict-prototypes -Wswitch-enum... yes
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wstrict-prototypes -Wswitch-enum -Wvariable-decl... no
      checking whether C compiler accepts  -Wextra -Warray-bounds -Wbad-function-cast -Wcast-qual -Wdiv-by-zero -Wduplicated-branches -Wduplicated-cond -Wfloat-equal -Wformat=2 -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-type-limits -Wno-unknown-pragmas -Wnormalized=id -Wnull-dereference -Wold-style-declaration -Wpointer-arith -Wredundant-decls -Wrestrict -Wstrict-prototypes -Wswitch-enum -Wwrite-strings... yes
      checking whether the linker accepts -Wl,-z,relro... yes
      checking whether the linker accepts -Wl,-z,now... yes
      checking whether the linker accepts -Wl,-z,noexecstack... yes
      checking whether segmentation violations can be caught... yes
      checking whether SIGABRT can be caught... no
      configure: WARNING: On this platform, SIGABRT cannot be caught using signal handlers.
      checking for thread local storage (TLS) class... thread_local
      thread local storage is supported
      checking whether C compiler accepts -ftls-model=local-dynamic... yes
      checking how to print strings... printf
      checking for a sed that does not truncate output... (cached) /usr/bin/sed
      checking for grep that handles long lines and -e... /usr/bin/grep
      checking for egrep... /usr/bin/grep -E
      checking for fgrep... /usr/bin/grep -F
      checking for ld used by gcc... /usr/bin/ld
      checking if the linker (/usr/bin/ld) is GNU ld... yes
      checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
      checking the name lister (/usr/bin/nm -B) interface... BSD nm
      checking whether ln -s works... yes
      checking the maximum length of command line arguments... 1572864
      checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
      checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
      checking for /usr/bin/ld option to reload object files... -r
      checking for file... file
      checking for objdump... objdump
      checking how to recognize dependent libraries... pass_all
      checking for dlltool... no
      checking how to associate runtime and link libraries... printf %s\n
      checking for ar... ar
      checking for archiver @FILE support... @
      checking for strip... strip
      checking for ranlib... ranlib
      checking command to parse /usr/bin/nm -B output from gcc object... ok
      checking for sysroot... no
      checking for a working dd... /usr/bin/dd
      checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
      checking for mt... no
      checking if : is a manifest tool... no
      checking for dlfcn.h... yes
      checking for objdir... .libs
      checking if gcc supports -fno-rtti -fno-exceptions... no
      checking for gcc option to produce PIC... -fPIC -DPIC
      checking if gcc PIC flag -fPIC -DPIC works... yes
      checking if gcc static flag -static works... no
      checking if gcc supports -c -o file.o... yes
      checking if gcc supports -c -o file.o... (cached) yes
      checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
      checking dynamic linker characteristics... GNU/Linux ld.so
      checking how to hardcode library paths into programs... immediate
      checking whether stripping libraries is possible... yes
      checking if libtool supports shared libraries... yes
      checking whether to build shared libraries... no
      checking whether to build static libraries... yes
      checking for ar... (cached) ar
      checking for ARM64 target... no
      checking whether C compiler accepts -mmmx... yes
      checking for MMX instructions set... yes
      checking whether C compiler accepts -mmmx... (cached) yes
      checking whether C compiler accepts -msse2... yes
      checking for SSE2 instructions set... yes
      checking whether C compiler accepts -msse2... (cached) yes
      checking whether C compiler accepts -msse3... yes
      checking for SSE3 instructions set... yes
      checking whether C compiler accepts -msse3... (cached) yes
      checking whether C compiler accepts -mssse3... yes
      checking for SSSE3 instructions set... yes
      checking whether C compiler accepts -mssse3... (cached) yes
      checking whether C compiler accepts -msse4.1... yes
      checking for SSE4.1 instructions set... yes
      checking whether C compiler accepts -msse4.1... (cached) yes
      checking whether C compiler accepts -mavx... yes
      checking for AVX instructions set... yes
      checking whether C compiler accepts -mavx... (cached) yes
      checking whether C compiler accepts -mavx2... yes
      checking for AVX2 instructions set... yes
      checking whether C compiler accepts -mavx2... (cached) yes
      checking if _mm256_broadcastsi128_si256 is correctly defined... yes
      checking whether C compiler accepts -mavx512f... yes
      checking for AVX512F instructions set... yes
      checking whether C compiler accepts -mavx512f... (cached) yes
      checking whether C compiler accepts -maes... yes
      checking whether C compiler accepts -mpclmul... yes
      checking for AESNI instructions set and PCLMULQDQ... yes
      checking whether C compiler accepts -maes... (cached) yes
      checking whether C compiler accepts -mpclmul... (cached) yes
      checking whether C compiler accepts -mrdrnd... yes
      checking for RDRAND... yes
      checking whether C compiler accepts -mrdrnd... (cached) yes
      checking for sys/mman.h... yes
      checking for sys/param.h... yes
      checking for sys/random.h... yes
      checking for intrin.h... no
      checking for sys/auxv.h... yes
      checking for CommonCrypto/CommonRandom.h... no
      checking for cet.h... yes
      checking if _xgetbv() is available... no
      checking for inline... inline
      checking whether byte ordering is bigendian... (cached) no
      checking whether __STDC_LIMIT_MACROS is required... no
      checking whether we can use inline asm code... yes
      no
      checking whether we can use x86_64 asm code... yes
      checking whether we can assemble AVX opcodes... yes
      checking for 128-bit arithmetic... yes
      checking for cpuid instruction... yes
      checking if the .private_extern asm directive is supported... no
      checking if the .hidden asm directive is supported... yes
      checking if weak symbols are supported... yes
      checking if atomic operations are supported... yes
      checking if C11 memory fences are supported... yes
      checking if gcc memory fences are supported... yes
      checking for size_t... yes
      checking for working alloca.h... yes
      checking for alloca... yes
      checking for arc4random... yes
      checking for arc4random_buf... yes
      checking for mmap... yes
      checking for mlock... yes
      checking for madvise... yes
      checking for mprotect... yes
      checking for raise... yes
      checking for sysconf... yes
      checking for getrandom with a standard API... yes
      checking for getrandom... yes
      checking for getentropy with a standard API... yes
      checking for getentropy... yes
      checking for getpid... yes
      checking for getauxval... yes
      checking for elf_aux_info... no
      checking for posix_memalign... yes
      checking for nanosleep... yes
      checking for clock_gettime... yes
      checking for memset_s... no
      checking for explicit_bzero... yes
      checking for memset_explicit... no
      checking for explicit_memset... no
      checking if gcc/ld supports -Wl,--output-def... not needed, shared libraries are disabled
      checking that generated files are newer than configure... done
      configure: creating ./config.status
      config.status: creating Makefile
      config.status: creating builds/Makefile
      config.status: creating contrib/Makefile
      config.status: creating dist-build/Makefile
      config.status: creating libsodium.pc
      config.status: creating libsodium-uninstalled.pc
      config.status: creating src/Makefile
      config.status: creating src/libsodium/Makefile
      config.status: creating src/libsodium/include/Makefile
      config.status: creating src/libsodium/include/sodium/version.h
      config.status: creating test/default/Makefile
      config.status: creating test/Makefile
      config.status: executing depfiles commands
      config.status: executing libtool commands
      'make'
      Making all in builds
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      make[1]: Nothing to be done for 'all'.
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      Making all in contrib
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      make[1]: Nothing to be done for 'all'.
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      Making all in dist-build
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      make[1]: Nothing to be done for 'all'.
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      Making all in src
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      Making all in libsodium
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      Making all in include
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[3]: Nothing to be done for 'all'.
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
        CC       crypto_aead/aegis128l/libsodium_la-aead_aegis128l.lo
        CC       crypto_aead/aegis128l/libsodium_la-aegis128l_soft.lo
        CC       crypto_aead/aegis256/libsodium_la-aead_aegis256.lo
        CC       crypto_aead/aegis256/libsodium_la-aegis256_soft.lo
        CC       crypto_aead/aes256gcm/libsodium_la-aead_aes256gcm.lo
        CC       crypto_aead/chacha20poly1305/libsodium_la-aead_chacha20poly1305.lo
        CC       crypto_aead/xchacha20poly1305/libsodium_la-aead_xchacha20poly1305.lo
        CC       crypto_auth/libsodium_la-crypto_auth.lo
        CC       crypto_auth/hmacsha256/libsodium_la-auth_hmacsha256.lo
        CC       crypto_auth/hmacsha512/libsodium_la-auth_hmacsha512.lo
        CC       crypto_auth/hmacsha512256/libsodium_la-auth_hmacsha512256.lo
        CC       crypto_box/libsodium_la-crypto_box.lo
        CC       crypto_box/libsodium_la-crypto_box_easy.lo
        CC       crypto_box/libsodium_la-crypto_box_seal.lo
        CC       crypto_box/curve25519xsalsa20poly1305/libsodium_la-box_curve25519xsalsa20poly1305.lo
        CC       crypto_core/ed25519/ref10/libsodium_la-ed25519_ref10.lo
        CC       crypto_core/hchacha20/libsodium_la-core_hchacha20.lo
        CC       crypto_core/hsalsa20/ref2/libsodium_la-core_hsalsa20_ref2.lo
        CC       crypto_core/hsalsa20/libsodium_la-core_hsalsa20.lo
        CC       crypto_core/salsa/ref/libsodium_la-core_salsa_ref.lo
        CC       crypto_core/softaes/libsodium_la-softaes.lo
        CC       crypto_generichash/libsodium_la-crypto_generichash.lo
        CC       crypto_generichash/blake2b/libsodium_la-generichash_blake2.lo
        CC       crypto_generichash/blake2b/ref/libsodium_la-blake2b-compress-ref.lo
        CC       crypto_generichash/blake2b/ref/libsodium_la-blake2b-ref.lo
        CC       crypto_generichash/blake2b/ref/libsodium_la-generichash_blake2b.lo
        CC       crypto_hash/libsodium_la-crypto_hash.lo
        CC       crypto_hash/sha256/libsodium_la-hash_sha256.lo
        CC       crypto_hash/sha256/cp/libsodium_la-hash_sha256_cp.lo
        CC       crypto_hash/sha512/libsodium_la-hash_sha512.lo
        CC       crypto_hash/sha512/cp/libsodium_la-hash_sha512_cp.lo
        CC       crypto_kdf/blake2b/libsodium_la-kdf_blake2b.lo
        CC       crypto_kdf/libsodium_la-crypto_kdf.lo
        CC       crypto_kdf/hkdf/libsodium_la-kdf_hkdf_sha256.lo
        CC       crypto_kdf/hkdf/libsodium_la-kdf_hkdf_sha512.lo
        CC       crypto_kx/libsodium_la-crypto_kx.lo
        CC       crypto_onetimeauth/libsodium_la-crypto_onetimeauth.lo
        CC       crypto_onetimeauth/poly1305/libsodium_la-onetimeauth_poly1305.lo
        CC       crypto_onetimeauth/poly1305/donna/libsodium_la-poly1305_donna.lo
        CC       crypto_pwhash/argon2/libsodium_la-argon2-core.lo
        CC       crypto_pwhash/argon2/libsodium_la-argon2-encoding.lo
        CC       crypto_pwhash/argon2/libsodium_la-argon2-fill-block-ref.lo
        CC       crypto_pwhash/argon2/libsodium_la-argon2.lo
        CC       crypto_pwhash/argon2/libsodium_la-blake2b-long.lo
        CC       crypto_pwhash/argon2/libsodium_la-pwhash_argon2i.lo
        CC       crypto_pwhash/argon2/libsodium_la-pwhash_argon2id.lo
        CC       crypto_pwhash/libsodium_la-crypto_pwhash.lo
        CC       crypto_scalarmult/libsodium_la-crypto_scalarmult.lo
        CC       crypto_scalarmult/curve25519/ref10/libsodium_la-x25519_ref10.lo
        CC       crypto_scalarmult/curve25519/libsodium_la-scalarmult_curve25519.lo
        CC       crypto_secretbox/libsodium_la-crypto_secretbox.lo
        CC       crypto_secretbox/libsodium_la-crypto_secretbox_easy.lo
        CC       crypto_secretbox/xsalsa20poly1305/libsodium_la-secretbox_xsalsa20poly1305.lo
        CC       crypto_secretstream/xchacha20poly1305/libsodium_la-secretstream_xchacha20poly1305.lo
        CC       crypto_shorthash/libsodium_la-crypto_shorthash.lo
        CC       crypto_shorthash/siphash24/libsodium_la-shorthash_siphash24.lo
        CC       crypto_shorthash/siphash24/ref/libsodium_la-shorthash_siphash24_ref.lo
        CC       crypto_sign/libsodium_la-crypto_sign.lo
        CC       crypto_sign/ed25519/libsodium_la-sign_ed25519.lo
        CC       crypto_sign/ed25519/ref10/libsodium_la-keypair.lo
        CC       crypto_sign/ed25519/ref10/libsodium_la-open.lo
        CC       crypto_sign/ed25519/ref10/libsodium_la-sign.lo
        CC       crypto_stream/chacha20/libsodium_la-stream_chacha20.lo
        CC       crypto_stream/chacha20/ref/libsodium_la-chacha20_ref.lo
        CC       crypto_stream/libsodium_la-crypto_stream.lo
        CC       crypto_stream/salsa20/libsodium_la-stream_salsa20.lo
        CC       crypto_stream/xsalsa20/libsodium_la-stream_xsalsa20.lo
        CC       crypto_verify/libsodium_la-verify.lo
        CC       randombytes/libsodium_la-randombytes.lo
        CC       sodium/libsodium_la-codecs.lo
        CC       sodium/libsodium_la-core.lo
        CC       sodium/libsodium_la-runtime.lo
        CC       sodium/libsodium_la-utils.lo
        CC       sodium/libsodium_la-version.lo
        CPPAS    crypto_stream/salsa20/xmm6/libsodium_la-salsa20_xmm6-asm.lo
        CC       crypto_stream/salsa20/xmm6/libsodium_la-salsa20_xmm6.lo
        CC       crypto_scalarmult/curve25519/sandy2x/libsodium_la-curve25519_sandy2x.lo
        CC       crypto_scalarmult/curve25519/sandy2x/libsodium_la-fe51_invert.lo
        CC       crypto_scalarmult/curve25519/sandy2x/libsodium_la-fe_frombytes_sandy2x.lo
        CPPAS    crypto_scalarmult/curve25519/sandy2x/libsodium_la-sandy2x.lo
        CC       crypto_box/curve25519xchacha20poly1305/libsodium_la-box_curve25519xchacha20poly1305.lo
        CC       crypto_box/curve25519xchacha20poly1305/libsodium_la-box_seal_curve25519xchacha20poly1305.lo
        CC       crypto_core/ed25519/libsodium_la-core_ed25519.lo
        CC       crypto_core/ed25519/libsodium_la-core_ristretto255.lo
        CC       crypto_pwhash/scryptsalsa208sha256/libsodium_la-crypto_scrypt-common.lo
        CC       crypto_pwhash/scryptsalsa208sha256/libsodium_la-scrypt_platform.lo
        CC       crypto_pwhash/scryptsalsa208sha256/libsodium_la-pbkdf2-sha256.lo
        CC       crypto_pwhash/scryptsalsa208sha256/libsodium_la-pwhash_scryptsalsa208sha256.lo
        CC       crypto_pwhash/scryptsalsa208sha256/nosse/libsodium_la-pwhash_scryptsalsa208sha256_nosse.lo
        CC       crypto_scalarmult/ed25519/ref10/libsodium_la-scalarmult_ed25519_ref10.lo
        CC       crypto_scalarmult/ristretto255/ref10/libsodium_la-scalarmult_ristretto255_ref10.lo
        CC       crypto_secretbox/xchacha20poly1305/libsodium_la-secretbox_xchacha20poly1305.lo
        CC       crypto_shorthash/siphash24/libsodium_la-shorthash_siphashx24.lo
        CC       crypto_shorthash/siphash24/ref/libsodium_la-shorthash_siphashx24_ref.lo
        CC       crypto_sign/ed25519/ref10/libsodium_la-obsolete.lo
        CC       crypto_stream/salsa2012/ref/libsodium_la-stream_salsa2012_ref.lo
        CC       crypto_stream/salsa2012/libsodium_la-stream_salsa2012.lo
        CC       crypto_stream/salsa208/ref/libsodium_la-stream_salsa208_ref.lo
        CC       crypto_stream/salsa208/libsodium_la-stream_salsa208.lo
        CC       crypto_stream/xchacha20/libsodium_la-stream_xchacha20.lo
        CC       randombytes/sysrandom/libsodium_la-randombytes_sysrandom.lo
        CC       crypto_aead/aegis128l/libaesni_la-aegis128l_aesni.lo
        CC       crypto_aead/aegis256/libaesni_la-aegis256_aesni.lo
        CC       crypto_aead/aes256gcm/aesni/libaesni_la-aead_aes256gcm_aesni.lo
        CCLD     libaesni.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_aead/aegis128l/libarmcrypto_la-aegis128l_armcrypto.lo
        CC       crypto_aead/aegis256/libarmcrypto_la-aegis256_armcrypto.lo
        CC       crypto_aead/aes256gcm/armcrypto/libarmcrypto_la-aead_aes256gcm_armcrypto.lo
        CCLD     libarmcrypto.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_onetimeauth/poly1305/sse2/libsse2_la-poly1305_sse2.lo
        CC       crypto_pwhash/scryptsalsa208sha256/sse/libsse2_la-pwhash_scryptsalsa208sha256_sse.lo
        CCLD     libsse2.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_generichash/blake2b/ref/libssse3_la-blake2b-compress-ssse3.lo
        CC       crypto_pwhash/argon2/libssse3_la-argon2-fill-block-ssse3.lo
        CC       crypto_stream/chacha20/dolbeau/libssse3_la-chacha20_dolbeau-ssse3.lo
        CCLD     libssse3.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_generichash/blake2b/ref/libsse41_la-blake2b-compress-sse41.lo
        CCLD     libsse41.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_generichash/blake2b/ref/libavx2_la-blake2b-compress-avx2.lo
        CC       crypto_pwhash/argon2/libavx2_la-argon2-fill-block-avx2.lo
        CC       crypto_stream/chacha20/dolbeau/libavx2_la-chacha20_dolbeau-avx2.lo
        CC       crypto_stream/salsa20/xmm6int/libavx2_la-salsa20_xmm6int-avx2.lo
        CCLD     libavx2.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       crypto_pwhash/argon2/libavx512f_la-argon2-fill-block-avx512f.lo
        CCLD     libavx512f.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CC       randombytes/internal/librdrand_la-randombytes_internal_random.lo
        CCLD     librdrand.la
      libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
        CCLD     libsodium.la
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[2]: Nothing to be done for 'all-am'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      Making all in test
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      Making all in default
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[2]: Nothing to be done for 'all'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[2]: Nothing to be done for 'all-am'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      make[1]: Nothing to be done for 'all-am'.
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      'make' 'install'
      Making install in builds
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      make[2]: Nothing to be done for 'install-exec-am'.
      make[2]: Nothing to be done for 'install-data-am'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/builds'
      Making install in contrib
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      make[2]: Nothing to be done for 'install-exec-am'.
      make[2]: Nothing to be done for 'install-data-am'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/contrib'
      Making install in dist-build
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      make[2]: Nothing to be done for 'install-exec-am'.
      make[2]: Nothing to be done for 'install-data-am'.
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/dist-build'
      Making install in src
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      Making install in libsodium
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      Making install in include
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[4]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[4]: Nothing to be done for 'install-exec-am'.
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/include'
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/include/sodium'
       /usr/bin/install -c -m 644  sodium/core.h sodium/crypto_aead_aes256gcm.h sodium/crypto_aead_aegis128l.h sodium/crypto_aead_aegis256.h sodium/crypto_aead_chacha20poly1305.h sodium/crypto_aead_xchacha20poly1305.h sodium/crypto_auth.h sodium/crypto_auth_hmacsha256.h sodium/crypto_auth_hmacsha512.h sodium/crypto_auth_hmacsha512256.h sodium/crypto_box.h sodium/crypto_box_curve25519xchacha20poly1305.h sodium/crypto_box_curve25519xsalsa20poly1305.h sodium/crypto_core_ed25519.h sodium/crypto_core_ristretto255.h sodium/crypto_core_hchacha20.h sodium/crypto_core_hsalsa20.h sodium/crypto_core_salsa20.h sodium/crypto_core_salsa2012.h sodium/crypto_core_salsa208.h sodium/crypto_generichash.h sodium/crypto_generichash_blake2b.h sodium/crypto_hash.h sodium/crypto_hash_sha256.h sodium/crypto_hash_sha512.h sodium/crypto_kdf.h sodium/crypto_kdf_hkdf_sha256.h sodium/crypto_kdf_hkdf_sha512.h sodium/crypto_kdf_blake2b.h sodium/crypto_kx.h sodium/crypto_onetimeauth.h sodium/crypto_onetimeauth_poly1305.h sodium/crypto_pwhash.h sodium/crypto_pwhash_argon2i.h sodium/crypto_pwhash_argon2id.h sodium/crypto_pwhash_scryptsalsa208sha256.h sodium/crypto_scalarmult.h sodium/crypto_scalarmult_curve25519.h sodium/crypto_scalarmult_ed25519.h sodium/crypto_scalarmult_ristretto255.h '/tmp/tmpziu6b28p/build/bundled/include/sodium'
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/include/sodium'
       /usr/bin/install -c -m 644  sodium/crypto_secretbox.h sodium/crypto_secretbox_xchacha20poly1305.h sodium/crypto_secretbox_xsalsa20poly1305.h sodium/crypto_secretstream_xchacha20poly1305.h sodium/crypto_shorthash.h sodium/crypto_shorthash_siphash24.h sodium/crypto_sign.h sodium/crypto_sign_ed25519.h sodium/crypto_sign_edwards25519sha512batch.h sodium/crypto_stream.h sodium/crypto_stream_chacha20.h sodium/crypto_stream_salsa20.h sodium/crypto_stream_salsa2012.h sodium/crypto_stream_salsa208.h sodium/crypto_stream_xchacha20.h sodium/crypto_stream_xsalsa20.h sodium/crypto_verify_16.h sodium/crypto_verify_32.h sodium/crypto_verify_64.h sodium/export.h sodium/randombytes.h sodium/randombytes_internal_random.h sodium/randombytes_sysrandom.h sodium/runtime.h sodium/utils.h '/tmp/tmpziu6b28p/build/bundled/include/sodium'
       /usr/bin/install -c -m 644  sodium.h '/tmp/tmpziu6b28p/build/bundled/include/.'
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/include'
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/include/sodium'
       /usr/bin/install -c -m 644  sodium/version.h '/tmp/tmpziu6b28p/build/bundled/include/sodium'
      make[4]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium/include'
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[4]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/lib'
       /bin/sh ../../libtool   --mode=install /usr/bin/install -c   libsodium.la '/tmp/tmpziu6b28p/build/bundled/lib'
      libtool: install: /usr/bin/install -c .libs/libsodium.lai /tmp/tmpziu6b28p/build/bundled/lib/libsodium.la
      libtool: install: /usr/bin/install -c .libs/libsodium.a /tmp/tmpziu6b28p/build/bundled/lib/libsodium.a
      libtool: install: chmod 644 /tmp/tmpziu6b28p/build/bundled/lib/libsodium.a
      libtool: install: ranlib /tmp/tmpziu6b28p/build/bundled/lib/libsodium.a
      libtool: finish: PATH="/tmp/pip-build-env-si4y4agd/overlay/bin:/tmp/pip-build-env-si4y4agd/normal/bin:/home/cc/venv/bin:/opt/gcc-latest/bin:/run/user/1000/fnm_multishells/6625_1742420752373/bin:/home/cc/.local/share/fnm:/home/cc/perl5/bin:/opt/gcc-latest/bin:/run/user/1000/fnm_multishells/4842_1742420645469/bin:/home/cc/.local/share/fnm:/home/cc/perl5/bin:/opt/gcc-latest/bin:/run/user/1000/fnm_multishells/4774_1742420645283/bin:/home/cc/.local/share/fnm:/home/cc/perl5/bin:/home/cc/.local/bin:/home/cc/bin:/home/cc/.cargo/bin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin" ldconfig -n /tmp/tmpziu6b28p/build/bundled/lib
      ----------------------------------------------------------------------
      Libraries have been installed in:
         /tmp/tmpziu6b28p/build/bundled/lib
      
      If you ever happen to want to link against installed libraries
      in a given directory, LIBDIR, you must either use libtool, and
      specify the full pathname of the library, or use the '-LLIBDIR'
      flag during linking and do at least one of the following:
         - add LIBDIR to the 'LD_LIBRARY_PATH' environment variable
           during execution
         - add LIBDIR to the 'LD_RUN_PATH' environment variable
           during linking
         - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
         - have your system administrator add LIBDIR to '/etc/ld.so.conf'
      
      See any operating system documentation about shared libraries for
      more information, such as the ld(1) and ld.so(8) manual pages.
      ----------------------------------------------------------------------
      make[4]: Nothing to be done for 'install-data-am'.
      make[4]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src/libsodium'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[3]: Nothing to be done for 'install-exec-am'.
      make[3]: Nothing to be done for 'install-data-am'.
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/src'
      Making install in test
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      Making install in default
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[3]: Nothing to be done for 'install-exec-am'.
      make[3]: Nothing to be done for 'install-data-am'.
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test/default'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[3]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[3]: Nothing to be done for 'install-exec-am'.
      make[3]: Nothing to be done for 'install-data-am'.
      make[3]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src/test'
      make[1]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      make[2]: Entering directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      make[2]: Nothing to be done for 'install-exec-am'.
       /usr/bin/mkdir -p '/tmp/tmpziu6b28p/build/bundled/lib/pkgconfig'
       /usr/bin/install -c -m 644 libsodium.pc '/tmp/tmpziu6b28p/build/bundled/lib/pkgconfig'
      make[2]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      make[1]: Leaving directory '/tmp/tmpziu6b28p/build/_deps/bundled_libsodium-src'
      -- Looking for kqueue
      -- Looking for kqueue - not found
      CMake Error at /tmp/tmpziu6b28p/build/_deps/bundled_libzmq-src/CMakeLists.txt:7 (cmake_minimum_required):
        Compatibility with CMake < 3.5 has been removed from CMake.
      
        Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
        to tell CMake that the project requires at least <min> but has been updated
        to work with policies introduced by <max> or earlier.
      
        Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
      
      
      -- Configuring incomplete, errors occurred!
      
      *** CMake configuration failed
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyzmq
Failed to build pyzmq                                                                                                        
ERROR: Failed to build installable wheels for some pyproject.toml based projects (pyzmq)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -303,6 +303,16 @@ if (ZMQ_PREFIX STREQUAL "bundled")
   endif()
 
   # use libzmq's own cmake, so we can import the libzmq-static target
+  # libzmq uses an ancient policy minimum, no longer supported
+  if ("${CMAKE_POLICY_VERSION_MINIMUM}" STREQUAL "")
+    # workaround https://github.com/zeromq/libzmq/pull/4776
+    if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.31")
+      set(CMAKE_POLICY_VERSION_MINIMUM "3.31")
+    else()
+      set(CMAKE_POLICY_VERSION_MINIMUM "${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}")
+    endif()
+    message(STATUS "Setting CMAKE_POLICY_VERSION_MINIMUM=${CMAKE_POLICY_VERSION_MINIMUM} for libzmq")
+  endif()
   set(ENABLE_CURVE ON)
   set(ENABLE_DRAFTS ${ZMQ_DRAFT_API})
   set(ENABLE_LIBSODIUM_RANDOMBYTES_CLOSE "OFF")

